OrbitQt/main.cpp

```diff
@@ -7,27 +7,31 @@
 #include <QFontDatabase>
 #include <QStyleFactory>
 
-#include "absl/flags/flag.h"
-#include "absl/flags/parse.h"
-#include "absl/flags/usage.h"
 
-#include "../OrbitGl/App.h"
+#include "App.h"
+#include "ApplicationOptions.h"
 #include "CrashHandler.h"
 #include "Path.h"
+#include "absl/flags/flag.h"
+#include "absl/flags/parse.h"
+#include "absl/flags/usage.h"
 #include "orbitmainwindow.h"
 
 // TODO: Remove this flag once we have a dialog with user
 ABSL_FLAG(bool, upload_dumps_to_server, false,
           "Upload dumps to collection server when crashes");
 
+ABSL_FLAG(std::string, gamelet, "",
```

Comment:
Would it make sense to just make this a more general "remote" option? This should all work when you just have any remote machine running OrbitService, right?

Reply:
Considering there is going to be grpc_remote remote soon, not sure if this should be exposed in naming.. my initial thinking was to have --gamelet for old string and --remote for the new one, but it seems like this is confusing.

Given that port is currently fixed - maybe have user provide only hostname is a better option, in which case we can have only one option for the remote.

Reply:
I don't really have an opinion on this, but maybe I can give some context how it will probably look like when we future integrated deployment of OrbitService:
- In the future it will be required to give the name of the gamelet here instead of the IP, because we need the name to ask the GGP-SDK for the corresponding authentication-keys.
- Additionally we can support general-purpose-SSH with or without automatic deployment with little additional effort. In this case the user wants to be able to give a username, a hostname and a port.

Reply:
I changed the name to "remote", I also made port optional, we will add the port if user has not specified one.

Also I do not think this is particularly important since we still want an option to do everything from the UI, which will override this option.

Reply:
Sounds good, thanks!